ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -188,11 +189,12 @@ module.exports = function CesiumMap(
   selectionInterface,
   notificationEl,
   componentElement,
-  mapModel
+  mapModel,
+  cameraOptions
 ) {
   let overlays = {}
   let shapes = []
-  const map = createMap(insertionElement)
+  const map = createMap(insertionElement, cameraOptions && cameraOptions.cesium)
```

Comment:
This is sending cameraOptions.cesium unless cameraOptions is null, in which case it sends null?

Then when used in createMap, it is cameraOptions?  So cameraOptions.cesium is the type expected by createMap?

Reply:
Yes to both questions.